spec/models/library_entry_spec.rb

```diff
@@ -34,7 +36,6 @@
 #  index_library_entries_on_user_id_and_media_type_and_media_id  (user_id,media_type,media_id) UNIQUE
 #  index_library_entries_on_user_id_and_status                   (user_id,status)
 #
-# rubocop:enable Metrics/LineLength
 
 require 'rails_helper'
 
```

Comment:
Just a general comment you can create multiple let values at the top with different names. This is where they should all be (starting below `RSpec.describe`)